src/app/market/buy/checkout-process/checkout-process.component.html

```diff
@@ -251,11 +251,11 @@
             {{listing.basePrice?.getIntegerPart()}}{{listing.basePrice?.dot()}}<small>{{listing.basePrice?.getFractionalPart()}}</small>
             PART
           </td>
-          <td class="shipping cell">{{listing.internationalShippingPrice?.getIntegerPart()}}{{listing.internationalShippingPrice?.dot()}}<small>{{listing.internationalShippingPrice?.getFractionalPart()}}</small>
+          <td class="shipping cell">{{listing?.shippingAmount(shippingFormGroup.value.country || '')?.getIntegerPart()}}{{listing?.shippingAmount(shippingFormGroup.value.country || '')?.dot()}}<small>{{listing?.shippingAmount(shippingFormGroup.value.country || '')?.getFractionalPart()}}</small>
```

Comment:
What about 
public country = this.shippingFormGroup.value.country || '';

and then just referencing the country variable, short and sweet.